docs/source/markdown/podman-info.1.md

```diff
@@ -87,6 +87,15 @@ host:
       spec: 1.0.0
       +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
   os: linux
+  pasta:
+    executable: /usr/bin/passt
+    package: passt-0^20221116.gace074c-1.fc35.aarch64
```

Comment:
Sigh. I'm really sorry, I really appreciate your updating the documentation.... but I think a casual reader might be confused by the mix of fc34/fc35 and amd64/aarch64 in this output. Could you massage it a little please? Change fc35 to fc34, and aarch64 to amd64 in all the new additions? (The output does not need to reflect True Reality™, it just needs to be internally consistent). Thank you!

Reply:
> Sigh. I'm really sorry, I really appreciate your updating the documentation.... but I think a casual reader might be confused by the mix of fc34/fc35 and amd64/aarch64 in this output. Could you massage it a little please? Change fc35 to fc34, and aarch64 to amd64 in all the new additions? (The output does not need to reflect True Reality™, it just needs to be internally consistent). Thank you!

All right